.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release
+
+on:
+  pull_request:
```

Comment:
This should be 

```yaml
on:
  push:
    branches:
      - master
```
Right? Because you want this to run once the PR is merged into master right? The current code would run this even when a PR to master was closed but not merged.

Reply:
Good catch, I totally oversaw that scenario!

Reply:
I've updated 6189c38 !